thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/datalayer/DaoTestUtils.java

```diff
@@ -275,22 +273,6 @@ public static DetectionStatusDTO getTestDetectionStatus(String dataset, long dat
     return detectionStatusDTO;
   }
 
-  public static AutotuneConfigDTO getTestAutotuneConfig(long functionId, long start, long end) {
-    AutotuneConfigDTO autotuneConfigDTO = new AutotuneConfigDTO();
-    autotuneConfigDTO.setFunctionId(functionId);
-    autotuneConfigDTO.setStartTime(start);
-    autotuneConfigDTO.setEndTime(end);
-    autotuneConfigDTO.setPerformanceEvaluationMethod(PerformanceEvaluationMethod.ANOMALY_PERCENTAGE);
-    autotuneConfigDTO.setLastUpdateTimestamp(DateTime.now().getMillis());
-    Map<String, String> config = new HashMap<>();
-    config.put("ConfigKey", "ConfigValue");
-    autotuneConfigDTO.setConfiguration(config);
-    Map<String, Double> performance = new HashMap<>();
-    performance.put(autotuneConfigDTO.getPerformanceEvaluationMethod().name(), 0.5);
-    autotuneConfigDTO.setPerformance(performance);
-    return autotuneConfigDTO;
-  }
-
   public static ClassificationConfigDTO getTestClassificationConfig(String name, List<Long> mainFunctionIdList,
```

Comment:
Not used.

Reply:
This can be removed in next PR with all the unused DAO objects.